index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Book List</title>
+</head>
+<body>
+    <h1>Awesome Books</h1>
+    <div id="bookList"></div>
+    <form id="bookForm" class="BookForm">
+        <input type="text" name="title" id="inputTitle" placeholder="Title"> <br>
+        <br>
+        <input type="text" name="author" id="inputAuthor" placeholder="Author"><br>
+        <br><br>
+<button type="submit" id="add">submit</button>
```

Comment:
- [ ] I would recommend changing the change the text of this button for `add` or `Add`, this would reflect better the purpose of this button 😸